plugin-server/tests/main/db.test.ts

```diff
@@ -631,46 +631,6 @@ describe('DB', () => {
                 version: 2,
             })
         })
-
-        describe('with caching', () => {
-            it('insertGroup() and updateGroup() update cache', async () => {
```

Comment:
Do you think this test is worth keeping if we changed it to calling `upsertGroup`, or no?

Reply:
I thought about this, but we don't have any tests for that yet and that seems rather low priority given our plans to move to kv store longer term